tests/chart/test_dag_server_statefulset.py

```diff
@@ -296,3 +296,26 @@ def test_dag_server_statefulset_with_probes_and_authproxy_enabled(self, kube_ver
         assert livenessProbe == c_by_name["auth-proxy"]["livenessProbe"]
         assert readinessProbe == c_by_name["sidecar-log-consumer"]["readinessProbe"]
         assert livenessProbe == c_by_name["sidecar-log-consumer"]["livenessProbe"]
+
+    def test_dag_server_service_account_with_template(self, kube_version):
+        """Test dag-server service account with template."""
+        docs = render_chart(
+            kube_version=kube_version,
+            values={
+                "gitSyncRelay": {
+                    "enabled": True,
+                    "serviceAccount": {
+                        "create": False,
+                        "name": "custom-{{ .Release.Name }}-dag-processor",
+                    },
```

Comment:
Shouldn't there be an assertion about there not being a service account in the returned doc?